airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -71,6 +72,8 @@ def __init__(  # pylint: disable=too-many-arguments
         write_stdout: bool,
         json_format: bool,
         json_fields: str,
+        host_field: str,
+        offset_field: str,
```

Comment:
So, we have a bit of a chicken and egg problem here.

The config is going to be added in 2.1.1, but the ES logger is now in a provider, and that provider could be installed against Airflow 2.1.0.

So I think we have to do this

```suggestion
        host_field: str = "host",
        offset_field: str = "offset",
```

Reply:
oh good call